pkg/sql/schemachanger/rel/database.go

```diff
@@ -170,6 +264,9 @@ func (t *Database) insert(v interface{}, es entityStore) (id int, err error) {
 		if idx.exists != 0 && !idx.exists.isContainedIn(e.attrs) {
 			continue
 		}
+		if typ.isSliceMemberType && !idx.isInverted {
```

Comment:
I'd appreciate some commentary on why we want to skip in such case even it's clear that slice member can only be inserted into inverted index. like the good one you have on line #377